index.html

```diff
@@ -1167,9 +1167,6 @@ <h1>Speakers</h1>
                                 <div class="image-holder">
                                     <img class="background-image" alt="Khalil AlMaawali" src="img/KhalilAlMaawali.jpg">
                                     <div class="hover-state text-center preserve3d">
-                                        <div class="social-links vertical-align">
-                                            <a target="default" href="http//fossc.om"><i class="icon fa fa-external-link"></i></a>
```

Comment:
you don't have to remove the link just make it: http://fossc.om or http://fossc.om/2017/